src/borg/testsuite/archiver/debug_cmds.py

```diff
@@ -68,7 +68,7 @@ def test_debug_id_hash_format_put_get_parse_obj(self):
         """Test format-obj and parse-obj commands"""
 
         self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
-        data = b"some data"
+        data = b"some data with repeated sequence 00000000000000000000"
```

Comment:
maybe just use `b"some data" * 100`.

that should compress well enough to avoid a fallback to NONE.